### PR TITLE
feat: alert on provider upload size limit violations

### PR DESCRIPTION
**Pull request by vados-cosmonic**
# Feature or Problem

Add an explicit warning when files that are too big are uploaded

# Related Issues

# Release Information
`next`

# Consumer Impact
Affects provider upload modal of frontend (`wasmcloud_host`), improving it with an explicit alert when files are too big.

# Testing

Tested on platform(s)
- [x] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

## Platforms

Built on platform(s)
- [x] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

## Manual Verification

Manually verified:

https://user-images.githubusercontent.com/123968127/221045900-2de9ebcf-d95b-414a-b676-c177fc5050f8.mp4

### CODE REVIEW

**Review by thomastaylor312 — COMMENTED** [review]
I think this looks all ok. But I want to have @brooksmtownsend or someone else familiar with live view take a look at it

**Comment by vados-cosmonic** [comment]
thanks for taking a look -- just fixed the pesky lint...

**Comment by vados-cosmonic** [comment]
Flaky test?

```

  1) test Performing standard operations on actors stop with zero count terminates all (HostCore.ActorsTest)
Error:      test/host_core/actors_test.exs:509
     Assertion with == failed
     code:  assert actor_count == 5
     left:  4
     right: 5
     stacktrace:
       test/host_core/actors_test.exs:523: (test)
```

**Comment by brooksmtownsend** [comment]
> @brooksmtownsend is this test known to flake?

Not that I'm aware of... but I'm re-running to make sure

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# This file works with ASDF (https://github.com/asdf-vm/asdf)
+elixir 1.13.4-otp-25
```
[the remark this answers is not in the document]
Comment (vados-cosmonic) [reply .tool-versions:2]: Ahhhh OK, so I picked this version from what was in the tests -- I'm going to remove it. Maybe it makes sense to mandate which version people should have installed at some point, but that point does not have to be now!

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**